src/modules/services_manager_service.rb

```diff
@@ -12,31 +12,77 @@ class ServicesManagerServiceClass < Module
     LIST_UNIT_FILES_COMMAND = 'systemctl list-unit-files --type service'
     LIST_UNITS_COMMAND      = 'systemctl list-units --all --type service'
     STATUS_COMMAND          = 'systemctl status'
-    IS_ACTIVE_COMMAND       = 'systemctl is-active'
+    # WTF, duplicated in Yast::Systemctl
     COMMAND_OPTIONS         = ' --no-legend --no-pager --no-ask-password '
     TERM_OPTIONS            = ' LANG=C TERM=dumb COLUMNS=1024 '
     SERVICE_SUFFIX          = '.service'
 
+    # Used by ServicesManagerServiceClass to keep data about an individual service.
+    # (Not a real class; documents the structure of a Hash)
+    #
+    # Why does this hash exist if we have Yast::SystemdServiceClass::Service?
+    class Settings < Hash
+      # @!method [](k)
+      #   @option k :enabled  [Boolean] service has been enabled
+      #   @option k :can_be_enabled [Boolean] service can be enabled/disabled by the user
+      #   @option k :modified [Boolean] service has been changed (got enabled/disabled)
+      #   @option k :active   [Boolean] The high-level unit activation state, i.e. generalization of SUB
+      #   @option k :loaded   [Boolean] Reflects whether the unit definition was properly loaded
+      #   @option k :description [String] English description of the service
+    end
+
+    # @return [Settings]
     DEFAULT_SERVICE_SETTINGS = {
-      :enabled        => false, # Whether the service has been enabled
-      :can_be_enabled => true,  # Whether the service can be enabled/disabled by the user
-      :modified       => false, # Whether the service has been changed (got enabled/disabled)
-      :active         => false, # The high-level unit activation state, i.e. generalization of SUB
-      :loaded         => false, # Reflects whether the unit definition was properly loaded
-      :description    => nil    # English description of the service
+      :enabled        => false,
+      :can_be_enabled => true,
+      :modified       => false,
+      :active         => nil,
+      :loaded         => false,
+      :description    => nil
     }
 
+    # FIXME: this is a mixture of LoadState (unit) and UnitFileState (unit file)
     module Status
+      # LoadState
       LOADED     = 'loaded'
+      # LoadState
```

Comment:
???

Reply:
Yes? What should I fix or explain?

Reply:
why `NOTFOUND = 'not=found'` is commented as LoadState? Can you a slightly explain it? like possible LoadState ( with link to it ) enum value?

Reply:
OK, the module comment says 
>     # FIXME: this is a mixture of LoadState (unit) and UnitFileState (unit file)

so is it clearer if I change that to
>     # FIXME: this is a mixture of
>     # LoadState (the LOAD column of systemctl list-units) and
>     # UnitFileState (STATE of systemctl list-unit-files)

It's a fixme, I did not want to shave too many yaks.